airbyte-integrations/connectors/source-bigquery/acceptance-test-config.yml

```diff
@@ -20,9 +21,9 @@ acceptance_tests:
         configured_catalog_path: "integration_tests/configured_catalog.json"
         empty_streams: []
 # TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
-#        expect_records:
-#          path: "integration_tests/expected_records.jsonl"
-#          exact_order: no
+      #  expect_records:
+      #    path: "integration_tests/expected_records.jsonl"
+      #    exact_order: no
   incremental: 
     bypass_reason: "This connector does not implement incremental sync"
```

Comment:
Could you please clarify, why we don't test the `incremental` functionality here? Is it intentional for now, basically?

Reply:
@bazarnov 
I was originally planning to do it in other integration tests that's why. Now I updated the acceptance tests to also include the incremental here https://github.com/artefactory/airbyte/pull/28/commits/26d29a27deb987ac1d86ec7aba9478d4c89dd148